sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -424,7 +424,7 @@ def to_bigquery(
         job_config: bigquery.QueryJobConfig = None,
         timeout: int = 1800,
         retry_cadence: int = 10,
-    ) -> Optional[str]:
+    ) -> str:
```

Comment:
This function seems to always return a string, so the optional type is wrong. Mypy was complaining about one of its usages.